.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -4,23 +4,15 @@ labels: 'needs review'
 body:
   - type: markdown
     attributes:
-      value: Thanks for taking the time to file a bug report! Please fill out this form as completely as possible.
+      value: Thanks for taking the time to file a bug report! If you are convinced that you have found a bug in Expo tools, this is the right place. Please fill out this entire form.
```

Comment:
Not sure about 'Expo tools' but can't think of anything better either.

Reply:
Good catch, maybe we can rephrase it as "Expo code", "Expo products" or just "Expo"?

Reply:
With the "Expo tools" naming I'm little bit afraid that people will confuse this with our internal tools, which we refer to as ExpoTools (`et`):
* https://github.com/expo/expo/tree/main/tools

Reply:
@Simek that's a good point. Your suggestion of "Expo code" appealed to me as well, as it implies issues are for suspected bugs with _Expo_ code, and not _somebody else's_ code, invoked by Expo tools (though not necessarily ExpoTools, haha).

Reply:
I think "in Expo tools" or "originating in Expo" are good options.